ReactiveCocoa.podspec.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "ReactiveCocoa",
+  "version": “2.5.2”,
```

Comment:
Don't use TextEdit - it has specific symbols in it's encoding

I suggest:
1. TextWrangler
2. SublimeText
3. vim

These quotes are not quotes and will fire an error on install